dm/ctl/master/source_table_schema.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package master
+
+import (
+	"github.com/pingcap/dm/dm/ctl/common"
+	"github.com/pingcap/dm/dm/pb"
+
+	"github.com/spf13/cobra"
+)
+
+// NewSourceTableSchemaCmd creates a SourceTableSchema command.
+func NewSourceTableSchemaCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "source-table-schema <task-name> <table-filter1> <table-filter2> ...",
+		Short: "manage or show source-table schema schemas",
+		RunE:  sourceTableSchemaList,
+	}
+	cmd.AddCommand(
+		newSourceTableSchemaUpdateCmd(),
+		newSourceTableSchemaDeleteCmd(),
+	)
+
+	return cmd
+}
+
+func sourceTableSchemaList(cmd *cobra.Command, args []string) error {
+	if len(args) < 3 {
+		return cmd.Help()
+	}
+	taskName := common.GetTaskNameFromArgOrFile(args[0])
+	sources, err := common.GetSourceArgs(cmd)
+	if err != nil {
+		return err
+	}
+	database := args[1]
+	table := args[2]
```

Comment:
we support more table-filter as described in usage

Reply:
in next PR